english/crazy/crazy.md

```diff
@@ -1,2 +1,7 @@
 Crazy man!
 
+Do you...
+
+[Run](../run/run.md)
+[Hide](..hide/hide-chk.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.